summertime/dataset/dataset_loaders.py

```diff
@@ -552,3 +725,175 @@ def _process_data(self, data: Dataset) -> Generator[SummInstance, None, None]:
             summ_instance = SummInstance(source=article, summary=abstract)
 
             yield summ_instance
+
+
+class MassivesummDataset(SummDataset):
```

Comment:
Why `MassivesummDataset` does not have the method of `def is_supported(self, language: str):`?

Reply:
this is because Massivesumm doesn't yet allow for the multiple language initialization like `XlsumDataset(["yoruba", "vietnamese"])` can be done (planning to add this later when I can!) 

Currently though, the datasets library automatically checks if a config keyword (a language) passed into the Massivesumm initialization is one of the supported configs, so that's why there's no need for is_supported here.